null_jobs_backend/settings.py

```diff
@@ -241,3 +242,53 @@
 
 # GOOGLE AUTH
 SITE_ID = 2
+
+LOG_DIR = os.path.join(BASE_DIR, "logs")
+
+if not os.path.exists(LOG_DIR):
+    os.makedirs(LOG_DIR)
+
+LOGGING = {
+    "version": 1,
+    "disable_existing_loggers": False,
+    "formatters": {
+        "custom_format": {
+            "format": "%(process)d - %(asctime)s - %(levelname)s - %(message)s\n%(stack_info)s",
```

Comment:
I don't think `process id` is required here, because it will be the same as long as the DJANGO server is running. Keep only the `time - level - message`